committing.rst

```diff
@@ -258,6 +258,41 @@ Developers can apply labels to GitHub pull requests).
 
 .. _cherry_picker.py: https://github.com/python/core-workflow/tree/master/cherry_picker
 
+.. _git_worktrees:
+
+Speeding up development workflow using git worktrees
+----------------------------------------------------
+
+When backporting changes to multiple versions, it's often useful to have more than one checkout
+of the codebase. This way, you can do things like running tests on one version
+while committing a patch on another version. For example: ::
+
+    $ git worktree add ~/cpython_3.6 origin/3.6
+    Preparing /home/user/cpython_3.6 (identifier cpython_3.61)
+    HEAD is now at 49731ef8d2 dict: add iterkeys aliases in 3.5
+
+    $ cd ~/cpython_3.6;
+    $ git cherry-pick 686c146b1ca1eb5c7ef058bb28772ac612a8ec1d
+    [bpo-31620 c48137b107] bpo-31620: have asyncio/queues not leak memory when you've exceptions during waiting
+    Date: Fri Sep 29 01:03:19 2017 +0000
+    files changed, 25 insertions(+)
+    create mode 100644 Misc/NEWS.d/next/Library/2017-10-06-04-35-31.bpo-31620.gksLA1.rst
+
+    $ ./python -m test -j3
+
+    $ git worktree add ~/cpython_2.7 origin/2.7                                 
+    Preparing /home/user/cpython_2.7 (identifier cpython_2.7)
+    HEAD is now at e37fefba8b Revert the last change:
+
+    $ cd ~/cpython_2.7
+
+    # make changes 
+    
+    $ git commit 
+
+A branch can only be checked out in one of the worktrees unless you 
+use the ``--force`` option with ``git checkout`` or ``git worktree add``.
```

Comment:
This could be turned around:  "If you want to checkout a branch in multiple worktrees, you have to use the ``--force`` option with ``git checkout`` or ``git worktree add``."